tests/Moq.Tests/CustomTypeMatchersFixture.cs

```diff
@@ -148,7 +150,7 @@ public void It_Is_works_with_custom_matcher()
 			var invocationCount = 0;
 			var mock = new Mock<IX>();
 			mock.Setup(m => m.Method(It.Is<IntOrString>((arg, _) => acceptableArgs.Contains(arg))))
-			    .Callback((object arg) => invocationCount++);
+				.Callback((object arg) => invocationCount++);
```

Comment:
Please undo this whitespace-only change. (It's unrelated to your PR, and the spaces are intentional as they line up the dots vertically.)